### PR TITLE
made buttons interactive

### PR DESCRIPTION
**Pull request by oz03-hub**
This PR adds animations to buttons. Currently if you hover over a button, the button will go up slightly, and when you click it will go down. The animations can be changed, let me know, try to keep it simple, another simple approach could be when you hover over a button it will get slightly bigger, when you click it will get slightly dark, etc.

### CODE REVIEW

File: src/frontend/src/components/Buttons.js [hunk 1]
```diff
@@ -5,20 +5,26 @@ const Buttons = ({
   onSaveAnnotation,
   onClearAnnotation,
 }) => {
+  const buttons = [
```
Comment (SergePogorelov) [inline src/frontend/src/components/Buttons.js:8]: I like the idea of ​​separating the buttons into a separate constant!)

**Review by SergePogorelov — APPROVED** [review]
I really like the way the button is pressed (go down), but I'm not sure if it should be go up when hovering (it's discussion question)... although I like that part too!)

**Comment by oz03-hub** [comment]
Yeah  maybe the up and down transformation was a bit much, in this commit there is no hover functionality, when the user presses the button it be slightly darker hinting it was just pressed

**Review by ginic — APPROVED** [review]
Thanks, @oz03-hub ! I really like the look of this, and it increases the usability of the application for sure. 

It would be good to switch to a component library like Sarah suggested eventually, as they provide many of these UI improvements. For example the buttons in Material UI are look really slick and provide many additional options, https://mui.com/material-ui/react-button/#loading-button. However, adding the component library is a lot of work, so this is a good alternative until we're ready for that.

**Review by cheerstopaula — APPROVED** [review]
Didn't get to see the hovering part of the commit, but the making the button darker when pressing it looks good to me! I like how the app is way more responsive with the last two PRs, great work @oz03-hub :3

**Pull request merged**